### ISSUE TITLE
String parameter annotation docs

### ISSUE THREAD

**Issue by brandomr** [jataware/dojo-docs#4]
@marshHawk4 could you write a brief doc (e.g. a paragraph) on how to annotate string parameters?

What I’m noticing is there are some inconsistencies which I think are due to whether the string is within another string.

For example, annotating this config file:

```
curl -H "api-key: CFA327856422688ADCD11182479C8435E9FA54FE7034AE468B84CAB5172582EE" \
        -H "Content-Type: application/json" \
        -X POST \
        -d '{"city":"Boise", "product":"eggs", "num_days": 7}' \
        https://grocerybear.com/getitems
```

I had to include the quotes so I highlighted `"Boise"` which turned into: `"city": {{ city }}` etc. I _think_ this is because it's within a single quoted string `'{"city":"Boise", "product":"eggs", "num_days": 7}'`

However, for directives on the CLI I’ve had to ignore the quotes so would just annotate `"Boise"` to `"city": "{{ city }}"` 

**Bottom line:** we need a clear set of instructions telling users exactly what they can/should expect when they annotate string parameters.

**Comment by marshHawk4** [comment]
This issue was due to extra quotes put around string values in the config rehydrate code. To fix this issue, and make sure the user experience for annotating strings is the same for config files and directives, we need to have the user annotate values excluding  quotes if there are any.

For example here is a directive that we need to annotate:
```
command = 'curl -H "api-key: CFA327856422688ADCD11182479C8435E9FA54FE7034AE468B84CAB5172582EE" \
        -H "Content-Type: application/json" \
        -X POST \
        -d '{"city":"Boise", "product":"eggs", "num_days": 7}' \
        https://grocerybear.com/getitems
```
The user will need to highlight **Boise** , **eggs** and **7** since those are the values that will be exposed parameters. 
The dehydrated version will look like this:
```
command = 'curl -H "api-key: CFA327856422688ADCD11182479C8435E9FA54FE7034AE468B84CAB5172582EE" \
        -H "Content-Type: application/json" \
        -X POST \
        -d '{"city":"{{city}}", "product":"{{product}}", "num_days": {{num_days}}}' \
        https://grocerybear.com/getitems'
```
**You can see we are preserving the quotes in the command.** 

How to update the code in DOJO to fix this? The code located at https://github.com/jataware/dojo/blob/de30c0e62b284cb9c1e44c089aa36ee0088f550f/dmc/dags/model-xform.py#L102 will need to be removed and replaced with a simple loop over parameters as seen here
https://github.com/jataware/dojo/blob/de30c0e62b284cb9c1e44c089aa36ee0088f550f/api/src/runs.py#L101 . 

Also we will need to fix any dehydrated config files that are saved on s3. Pythia is one that comes to mind that has a config file that will break because of this change. We will need to add back in quotes around  the string values. (ie. fertilizer, management_practice ...)
